DbMocker/Data/MockDbParameter.cs

```diff
@@ -75,6 +78,11 @@ public override object Value
             set
             {
                 _value = value;
+
+                //set value for out parameter (dynamicParameter dapper)
+                if (_parameter != null)
+                    _parameter.Value = value; 
```

Comment:
I caught exceptions when using Dapper.DynamicParameters. 
I think it's happening because the Value's reference doesn't correlate between an instances DbParameter